### DISCUSSION TITLE
PRIMENG is not converting design  to code in FIGMA

### DISCUSSION THREAD

**Discussion by lucianatorres** [primefaces/community#1618] [PrimeNG]
Hey guys, how are you? I am new here, so I am sorry if I say something stupid. I am creating a design in Figma and I decided to buy UI KIT from Primeng because it was supposed to create a specific code for that design, but it is not working. When I ask to generate code, the outcome is not generating primeng tags components, classes, or attributes. Please help me.

**Comment by onursenture** [comment]
The feature you're requesting can be achieved with codegen plugins that operate in the Code tab of Dev Mode in Figma. However, the PrimeOne Figma UI Kit does not currently support such plugins, and this feature is not currently in our near-term plans.

However, we are continuing our work on a theme generator plugin that will enable design and development teams to establish a smoother workflow.
